src/frontend/app/store/actions/service-instances.actions.ts

```diff
@@ -0,0 +1,62 @@
+import { RequestOptions, URLSearchParams } from '@angular/http';
+import { schema } from 'normalizr';
+
+import { getAPIResourceGuid } from '../selectors/api.selectors';
+import { PaginationAction } from '../types/pagination.types';
+import { CFStartAction, ICFAction } from '../types/request.types';
+import { getActions } from './action.helper';
+import { ServiceInstancesSchema } from './action-types';
+
+export class GetServicesInstancesInSpace extends CFStartAction implements PaginationAction {
+  constructor(public cfGuid: string, public spaceGuid: string, public paginationKey: string) {
+    super();
+    this.options = new RequestOptions();
+    this.options.url = `spaces/${spaceGuid}/service_instances`;
+    this.options.method = 'get';
+    this.options.params = new URLSearchParams();
+  }
+  actions = getActions('Service Instaces', 'Get all in Space');
+  entity = [ServiceInstancesSchema];
+  entityKey = ServiceInstancesSchema.key;
+  options: RequestOptions;
+  initialParams = {
+    page: 1,
+    'results-per-page': 100,
```

Comment:
If this is used in a list it needs order-direction-field + order-direction for initial sorting values